neptune/new/handler.py

```diff
@@ -91,6 +91,18 @@ def upload_files(self, value: Union[str, Iterable[str]], wait: bool = False) ->
             else:
                 attr.upload_files(value, wait)
 
+    def fetch_values(self, include_timestamp=True):
+        with self._experiment.lock():
+            attr = self._experiment.get_attribute(self._path)
+            if attr:
+                if isinstance(attr, StringSeries):
+                    return attr.fetch_values(include_timestamp=include_timestamp)
+                elif isinstance(attr, FloatSeries):
+                    return attr.fetch_values(include_timestamp=include_timestamp)
```

Comment:
This command and command two lines above looks exactly the same, all this instance check is redundant.

Additionally we could use common [abstract class](https://realpython.com/python-interface/) for both `StringSeries` and `FloatSeries` to keep analogical interface for all Series.